src/texdocdialog.h

```diff
@@ -38,6 +39,7 @@ private slots:
 	QString lastDocRequest;
     Help *help;
     QStringList m_packages;
+    QButtonGroup ButtonGroup;

```

Comment:
to follow (implicit) naming strategy, the variable should start with a lower case letter (buttonGroup or even m_buttonGroup)

Reply:
Good point. I will review the code again to make these changes.